examples/src/bin/embassy_usb_serial.rs

```diff
@@ -0,0 +1,132 @@
+//! CDC-ACM serial port example using embassy.
+//!
+//! This example should be built in release mode.
+
+//% CHIPS: esp32s2 esp32s3
+//% FEATURES: async embassy embassy-executor-thread embassy-time-timg0 embassy-generic-timers
+
+#![no_std]
+#![no_main]
+#![feature(type_alias_impl_trait)]
```

Comment:
```suggestion
```

Reply:
Good eyes, thanks, it snuck in from some other example and I didn't remove it when I got rid of `make_static!`